cfgov/v1/management/commands/delete_page.py

```diff
@@ -0,0 +1,43 @@
+from django.core.exceptions import MultipleObjectsReturned
+from django.core.management.base import BaseCommand, CommandError
+
+from v1.models import CFGOVPage
+from v1.wagtail_hooks import flush_akamai
+
+
+class Command(BaseCommand):
+    help = 'Deletes a page by its slug name'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--slug',
+            nargs='?',
+            help='The slug for the page you wish to delete'
+        )
+        parser.add_argument(
+            '--id',
+            nargs='?',
+            help='The ID for the page you wish to delete'
+        )
+
+    def handle(self, *args, **options):
+        page_to_delete = None
+        slug = options['slug']
+        page_id = options['id']
+        if slug:
+            try:
+                page_to_delete = CFGOVPage.objects.get(slug=slug)
+            except MultipleObjectsReturned as e:
+                self.stderr.write(str(e))
+                raise CommandError('Slug `{}` did not work'.format(slug))
+        elif page_id:
+            page_to_delete = CFGOVPage.objects.get(id=page_id)
+        else:
+            raise CommandError('Must supply a slug or an id')
+        try:
+            page_to_delete.delete()
+            flush_akamai()
```

Comment:
This will flush Akamai a second time within this command, because Wagtail unpublishes before it deletes, and we have a hook to flush when unpublishing. However, since we need to flush the site _after_ we delete, we must call it explicitly. This is not ideal since we're flushing the entire site each time we flush, so we'll need to come back to this at some point.